src/utils/sourcemaps.rs

```diff
@@ -323,7 +309,12 @@ impl SourceMapProcessor {
                 file.contents.clear();
                 (SourceFileType::MinifiedSource, None)
             } else {
-                (SourceFileType::Source, None)
+                (
+                    SourceFileType::MinifiedSource,
```

Comment:
This variant should just be called `Source` now that it covers source files in general.

Reply:
The `SourceFileType` is defined in an external crate, so we cannot eliminate `MinifiedSource` in this PR, anyways. We additionally appear to be continuing to use both the `Source` and `MinifiedSource` variants in other unrelated areas of the CLI codebase. Although I agree that changing this to use `Source` would make sense, I think using `MinifiedSource` ensures that we apply the logic that we are currently applying to minified source files to all JS source files going forward. If we use `Source` instead, we would need to update any other code in this code path to treat `Source` the way we currently treat `MinifiedSource`, which would make this change more complex.

Since my goal with this PR is to provide a quick fix for the bug, and since I intend to refactor and simplify all of this code in the future, I think it is okay for us to continue using `MinifiedSource` in this PR.

Reply:
Are you okay with keeping this as `MinifiedSource` given the reasoning I have shared above, @loewenheim?

Reply:
Yes, but perhaps leave a comment.